src/audio/src/src.c

```diff
@@ -624,12 +636,22 @@ static void src_free(struct comp_dev *dev)
 	rfree(dev);
 }
 
+static int src_stream_pcm_source_rate_check(struct ipc_config_src cfg,
+					    struct sof_ipc_stream_params *params)
+{
+	if (cfg.source_rate && params->rate != cfg.source_rate)
+		return -EINVAL;
+
+	return 0;
+}
```

Comment:
could you keep this function where it was to reduce the diff?

Reply:
> 

Thanks‘s for suggestion, I have changed code.

Reply:
updated